.zenodo.json

```
@@ -35,6 +35,16 @@
             "affiliation": "Lawrence Berkeley National Laboratory",
             "name": "Zhang, Weiqun",
             "orcid": "0000-0001-8092-1974"
+        },
+        {
+            "affiliation": "Deutsches Elektronen-Synchrotron",
+            "name": "Hui, Xingjian",
+            "orcid": "0009-0005-8929-7318"
+        },
+        {
+            "affiliation": "Deutsches Elektronen-Synchrotron",
+            "name": "Dammak, Eya",
+            "orcid": "0000-0000-0000-0000"
```

```suggestion
            "name": "Dammak, Eya"
```
It also works without orcid, as can be seen with a few people on the WarpX zenodo page https://zenodo.org/records/16750069

---

I've just created one as I needed it: 0009-0003-7821-3508